erpy/instances/loggers/wandb_logger.py

```diff
@@ -57,6 +59,9 @@ def wandb_log_unknown(run: WandBRun, name: str, data: Any, step: int) -> None:
 
 
 class WandBLogger(Logger):
+
+    run: Run | RunDisabled | None
```

Comment:
Erpy is written in Python 3.8 & doesn't support using the pipe operator for union types. This is also why your previous PR's still used `Optional` to indicate optional types.